client/src/components/AddNewProfile.js

```diff
@@ -12,18 +13,16 @@ const AddNewProfile = () => {
     phone: "",
     gender: "",
     groups: "",
-    support_type: ""
+    support_type: "",
+    profile_type: "",
+    status: "",
+    join_date: ""
```

Comment:
```suggestion
    status: "new",
    join_date: new Date(),
```
We pass into `useState` what the initial values of our data should be, so it's here we should set the default values for `status` and `join_date`. This means when we click the `Add new profile` button, these values will be sent to the backend even if we didn't touch the dropdown and date picker.

We also need to connect these values to the dropdowns and date picker by adding `value={profileData.profile_type}`, `value={profileData.status}` and `value={profileData.join_date}` to them.